front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -23,9 +26,14 @@ type MapSearchStationProps = {
 const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchStationProps) => {
   const map = useSelector(getMap);
   const [searchState, setSearch] = useState<string>('');
+  const [chCodeFilter, setChCodeFilter] = useState<string>('');
+  const [bvOnly, setBvOnly] = useState<boolean>(false);
   const [searchResults, setSearchResults] = useState<
     SearchResultItemOperationalPoint[] | undefined
   >(undefined);
```

Comment:
I don't think we need the undefined possibility. We could use empty array as default value and then set the state back to that when clearing the input and finally you won't have to use the `.?` for the results count.

```suggestion
  const [searchResults, setSearchResults] = useState<
    SearchResultItemOperationalPoint[]
  >([]);
```